packages/amplify-graphql-model-transformer/package.json

```diff
@@ -44,6 +44,7 @@
         "@aws-cdk/custom-resources": "~1.124.0",
         "@aws-cdk/cx-api": "~1.124.0",
         "@aws-cdk/region-info": "~1.124.0",
+        "amplify-cli-core": "^2.5.0",
```

Comment:
IMO we shouldn't introduce dependency on CLI core for GraphQL transformers.

Reply:
updated